fivetran_provider_async/operators.py

```diff
@@ -21,8 +22,10 @@ class RegistryLink(BaseOperatorLink):
     def get_link(self, operator, dttm):
         """Get link to registry page."""
 
-        registry_link = "https://registry.astronomer.io/providers/{provider}/modules/{operator}"
-        return registry_link.format(provider="fivetran", operator="fivetranoperator")
+        return (
+            f"https://registry.astronomer.io/providers/airflow-provider-fivetran-async/versions/"
+            f"{__version__}/modules/FivetranOperator"
```

Comment:
```suggestion
            f"{__version__}/modules/{operator}"
```
we can check if this works